libmscore/read206.cpp

```diff
@@ -2064,6 +2064,16 @@ bool readChordRestProperties206(XmlReader& e, ChordRest* ch)
             QPointF pt = e.readPoint();
             ch->setOffset(pt * ch->spatium());
             }
+      else if (tag == "Image"){
```

Comment:
Maybe it would be better to check also that this is actually a rest (something like `ch->isRest() && tag == "Image"`), `Chord::add()` function doesn't support adding images anyway.